### DISCUSSION TITLE
[Suggestion] Correct blog post around behavior of 3.13 and CQv2

### DISCUSSION THREAD

**Discussion by HakShak** [rabbitmq/rabbitmq-server#12330] [Ideas] [answered]
### RabbitMQ series

3.13.x

### Operating system (distribution) used

linux

### How is RabbitMQ deployed?

Community Docker image

### What would you like to suggest for a future version of RabbitMQ?

https://www.rabbitmq.com/blog/2023/05/17/rabbitmq-3.12-performance-improvements#classic-queues-massively-improved-classic-queues-v2-cqv2

> Version 2 will become the default in 3.13 and will become the only implementation later on.

We can't find any documentation that corroborates this and it caused a bit of confusion.

After testing on a newly upgrades 3.13 cluster from just using the admin UI to make a new queue, it reports that it is CQv1 by default.

**Comment by mkuratczyk** [comment, marked as answer]
Yes, we were planning on making it the default in 3.13 but ultimately decided against it. I thought we updated the post, but maybe not. Anyway, I will fix this, thanks.

CQv2 becomes the only option in RabbitMQ 4.0. Any CQv1 will be automatically migrated if you upgrade to 4.0. I'd recommend switching to CQv2 on 3.13 (you will get the benefits of v2 and a simpler upgrade to 4.0)
